### PR TITLE
Deal with IE-only Range related features

### PR DESCRIPTION
**Pull request by teoli2003**
One is the interface, and the two others belong to `Element` according to MS docs. (Part of #16255)

### CODE REVIEW

File: files/en-us/web/api/msrangecollection/index.md [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+page-type: api-interface
 tags:
   - MSRangeCollection
+  - Non-standard
+  - Interface
 ---
 {{non-standard_header}}{{APIRef}}
 
-The **`MSRangeCollection`** object is an array of one or more Range objects.
+The **`MSRangeCollection`** interface is an array of one or more Range objects.
```
Comment (wbamberg) [inline files/en-us/web/api/msrangecollection/index.md:12]: I thought we could delete this one, as it's just an array of Range objects, and in the one place we refer to it, say "an array of Range objects" instead. Meaning, this is more like a typedef than an interface.

Comment (teoli2003) [reply files/en-us/web/api/msrangecollection/index.md:12]: Oh yes let's do this.

**Comment by teoli2003** [comment]
I have them conflicts but I can't see them… What should I do 🥵 ?

**Comment by wbamberg** [comment]
I thought this might happen, with all these moves. I guess they are in redirects and wikihistory. I reckon it might be easier to do these PRs one at a time. Sorry :(

**Comment by teoli2003** [comment]
Yeah I found them, they are a bit annoying. I'll try to fix them later (no awake enough)

**Review by wbamberg — APPROVED** [review]
👍 thank you @teoli2003 !

**Pull request merged**